doc/rst/source/explain_-q_full.rst_

```diff
@@ -0,0 +1,13 @@
+.. _-q_full:
+
+**-q**\ [**i**\ \|\ **o**][~]\ *rows*\ [**+c**\ *col*][**+a**\ \|\ **f**\ \|\ **s**]
+    Select specific data rows to be read (**-qi** [Default]) or written (**-qo**) [all].
+    Give individual rows (or row ranges in the format *start*\ [:*inc*]:*stop*, where
+    *inc* defaults to 1 if not specified) separated by commas [Default reads all rows,
+    starting with the first row (0)]. By default (**+a**) we count rows in the data set;
+    append **+f** or **+s** to reset the count at the start of each file or segment.
+    Alternatively, use **+c**\ *col* to indicate that the arguments instead
+    are *data limits* for monotonically increasing *col* values (such as in a time series).
```

Comment:
Why do the data must be "monotonically increasing"?

Reply:
They don't, I will update.